dataset/batch.py

```diff
@@ -271,7 +272,7 @@ def __setstate__(self, state):
 
     @property
     def _empty_data(self):
-        return None if self.components is None else self._item_class()   # pylint: disable=not-callable
+        return None if self.components is None else self._item_class()  # pylint: disable=not-callable
```

Comment:
Removing spaces is hardly necessary and should be avoided unless it clearly breaks PEP8.